queries.sql

```diff
@@ -10,3 +10,41 @@ SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name != 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
 
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
```

Comment:
- [ ] The end result of this transaction should assign the name `Digimon` as a specie to all records with a name that ends with `mon` while the rest of the entries should be assigned the name `Pokemon` as a specie. Because you had assigned some entries species names in `data.sql`, this change is not taking effect, kindly fix this so that the correct result shows
![image](https://user-images.githubusercontent.com/81932478/235707468-68928aaa-d644-4b3a-9b4b-93aa63f6a9e3.png)